data-tck/src/main/groovy/io/micronaut/data/tck/tests/AbstractRepositorySpec.groovy

```diff
@@ -1532,15 +1536,15 @@ abstract class AbstractRepositorySpec extends Specification {
         then:
         author.getBooks().size() == 4
 
-        when:
-        authorRepository.delete(author)
-        then:
-        author.getBooks().size() == 4
-        author.getBooks()[0].postLoad == 1
-        author.getBooks()[0].prePersist == 0
-        author.getBooks()[0].postPersist == 0
-        author.getBooks()[0].preUpdate == 0
-        author.getBooks()[0].postUpdate == 0
+//        when:
+//        authorRepository.delete(author)
```

Comment:
This is one example of potential issues. Adding FKs doesn't let us test this because records can be deleted since Author depends on Book and Book on Page, Chapter etc. so this test was able to delete Author without deleting child entitties and now is not because of FKs.